src/main/resources/application.yaml

```diff
@@ -24,6 +24,7 @@ security:
   jwks-uri: ${SSO_HOST}/auth/realms/fdk/protocol/openid-connect/certs
 
 application:
+  frontend-url: ${DATASERVICE_CATALOG_GUI_URL}
```

Comment:
Use fallback here instead of duplicating the property further down?

Reply:
I'm not sure we should use fallbacks in prod environment. Easier to detect bad config if it spits out an error, rather than fallback silently.